### ISSUE TITLE
Cannot find plugins

### ISSUE THREAD

**Issue by rmorrin** [lennybacon/KarmaVS#7]
I'm getting a warning when trying to use the karma-jasmine plugin:

```
INFO: Configuration found: {PATH OMITTED}\karma.unit.conf.js
INFO: Starting karma server...
WARN [plugin]: Cannot find plugin "karma-jasmine".
  Did you forget to install it ?
  npm install karma-jasmine --save-dev
```

All output stops after this stage. Any ideas? Everything works fine when running from the command line.

Thanks
Ryan

**Comment by lennybacon** [comment]
How did you start from the command line. In KarmaVS its done here: https://github.com/lennybacon/KarmaVS/blob/master/src/KarmaVs/KarmaVsPackage.cs#L271

**Comment by rmorrin** [comment]
Currently using Grunt to watch files and trigger tests on change, but if I switch to my web project's directory and run `karma start karma.unit.conf.js` manually everything works as expected.

Looks to be what the extension is doing too?

**Comment by lennybacon** [comment]
Can you check the environment variables in bot environments and compare them?

**Comment by rmorrin** [comment]
I can't see anything obviously different. The tests also work if I try the same karma commands in the package manager console. Does the extension run in this same environment?

**Comment by lennybacon** [comment]
Can you start karma from the package manager console inside visual studio?

**Comment by rmorrin** [comment]
Yes, tests are working correctly from the package manager console. I've included a copy of the output if that helps at all.

``` shell
PM> karma start --no-colors .\{PATH OMITTED}\karma.unit.conf.js
INFO [karma]: Karma v0.12.6 server started at http://localhost:9876/
INFO [launcher]: Starting browser PhantomJS
INFO [PhantomJS 1.9.7 (Windows 8)]: Connected on socket vM4jv-o_Pbh_jlCTMlXF with id 691523

...Tests Omitted...

PhantomJS 1.9.7 (Windows 8): Executed 30 of 30 SUCCESS (0.119 secs / 0.115 secs)
```
